azplugins/bond.py

```diff
@@ -148,19 +148,21 @@ def __init__(self, name=None):
         hoomd.context.current.system.addCompute(self.cpp_force, self.force_name)
 
         # setup the coefficient options
-        self.required_coeffs = ['k','r0','epsilon','sigma']
+        self.required_coeffs = ['k','r0','epsilon','sigma','delta']
+        self.bond_coeff.set_default_coeff('delta', 0.0)
 
     def process_coeff(self, coeff):
         k = coeff['k']
         r0 = coeff['r0']
         epsilon = coeff['epsilon']
         sigma = coeff['sigma']
+        delta = coeff['delta']
         lj1 = 4.0 * epsilon * math.pow(sigma, 12.0)
         lj2 = 4.0 * epsilon * math.pow(sigma, 6.0)
 
-        if epsilon==0:
-            hoomd.context.msg.error("azplugins.bond.fene(): epsilon must be non-zero.\n")
-            raise ValueError('epsilon must be non-zero')
+        #if epsilon==0:
```

Comment:
Was there are a reason this got commented out? Please revert, and we'll fix if there are other issues.

Reply:
> 

Hi Mike,
Just a question regarding this. Why do you want to raise an error when epsilon==0? I mean in WCA or in LJ we should be able to put epsilon=0 right? I remember in HOOMD's fene bonds we could set it to zero, so I thought may be we should have the same thing here. I wanted to ask you about this before hand, but I think I just forgot. Please let me know why we have to raise such an error here.

Reply:
I would really like FENE bonds to handle epsilon =0 case, because I think it would be more ideal for my system. I also would like to know whether these potentials (both pair and bonds ) in azplugins can be enabled and disabled just like in HOOMD?

Reply:
Thank you, that would really be helpful. If you can let me know what all to  change, I would try to implement the changes as fast I can.